packages/backend/src/Tabulators/AllocatedScore.ts

```diff
@@ -374,17 +373,17 @@ function normalizeArray(scores: ballot[], maxScore: number) {
 }
 
 function findSplitPoint(cand_df_sorted: winner_scores[], quota: typeof Fraction) {
-    var under_quota : any[] = [];
-    var under_quota_scores: typeof Fraction[] = [];
-    var cumsum = new Fraction(0);
-    cand_df_sorted.forEach((c, i) => {
+    let cumsum : typeof Fraction = new Fraction(0);
+    for(const c of cand_df_sorted ){
         cumsum = cumsum.add(c.ballot_weight);
-        if (cumsum < quota || i == 0) {
```

Comment:
We should add a test case for this i == 0 scenario. I think it was added when the number of votes is less that number of seats so the first voter will always exceed the threshold.

Reply:
Added!